tests/test_expenses.py

```diff
@@ -21,11 +21,16 @@ def test_delete_expense(self, generate_expense):
 def _helper_create_houses():
     house_1 = HouseFactory(user=UserFactory())
     house_2 = HouseFactory(user=UserFactory())
-    Expenses(house_name=house_1, amount=100, date=DATE, category=Expenses.Category.FOOD).save()
-    Expenses(house_name=house_1, amount=200, date=DATE, category=Expenses.Category.FOOD).save()
-    Expenses(house_name=house_2, amount=300, date=DATE, category=Expenses.Category.FOOD).save()
-    Expenses(house_name=house_2, amount=300, date=DATE, category=Expenses.Category.KIDS).save()
-    Expenses(house_name=house_2, amount=400, date=DATE, category=Expenses.Category.KIDS).save()
+    Expenses(house_name=house_1, amount=100, date=DATE, category=Expenses.Category.FOOD, description="description")\
+        .save()
+    Expenses(house_name=house_1, amount=200, date=DATE, category=Expenses.Category.FOOD, description="description")\
+        .save()
+    Expenses(house_name=house_2, amount=300, date=DATE, category=Expenses.Category.FOOD, description="description")\
+        .save()
+    Expenses(house_name=house_2, amount=300, date=DATE, category=Expenses.Category.KIDS, description="description")\
+        .save()
+    Expenses(house_name=house_2, amount=400, date=DATE, category=Expenses.Category.KIDS, description="description")\
```

Comment:
You should have bad data checks here as well, this is the place you are checking the expense creation.

Reply:
fixed